pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -113,6 +114,8 @@ stdenv.mkDerivation ({
 
   postPatch = ''
     patchShebangs make
+    substituteInPlace lib/os_mon/src/disksup.erl \
+      --replace '"sh ' '"${bash}/bin/sh '
```

```suggestion
    substituteInPlace lib/os_mon/src/disksup.erl \
      --replace '"sh ' '"${bash}/bin/sh '
```
Currently `substituteInPlace` has `--replace` deprecated.
Suggested usage: `--replace-fail`, `--replace-warn` and `--replace-quiet`

---

> This issue only started happening in OTP 26, is generic-builder.nix the best place for it?

Considering this, will it fail for other Erlang versions? Humm...